src/CommentList.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Comment from './Comment';
+
+const CommentList = ({ findcommet }) => (
+  <section className="commentSection">
+    {
+      findcommet.map(comment => (
+        <Comment key={Math.random() * 1000} comment={comment} />))
```

Comment:
```suggestion
        <Comment key={Math.random() * 1000} comment={comment} />))
```
Це унікальне значення ключа ти певен в цьому?